src/main/java/net/fabricmc/loom/task/GenerateSourcesTask.java

```diff
@@ -102,4 +105,17 @@ private void remapLineNumbers(Path oldCompiledJar, Path linemap, Path linemapped
 
 		progressLogger.completed();
 	}
+
+	private static File getMappedByproduct(Project project, String suffix) {
```

Comment:
mapped byproduct? this doesn't make sense to someone who doesn't really know all the code surrounding this